calculator.rb

```diff
@@ -0,0 +1,83 @@
+# Calculator Project - 8.8.18
+
+# print welcome message and operator options
+puts "Welcome to The Calculator! Which operator would you like to use?"
+puts "1. add(+)"
+puts "2. subtract(-)"
+puts "3. multiply(*)"
+puts "4. divide(/)"
+
+# user input - operation
+puts "Please choose one operator. You can enter the name or the symbol."
+op = gets.chomp
+
+valid_ops = %w[add + subtract - multiply * divide /]
+
+# check for valid user input - operation
+until valid_ops.include?(op)
+  puts "Invalid input."
+  puts "Please choose one operator. You can enter the name or the symbol."
+  op = gets.chomp
+end
+
+# method to verify user input - integer
+def integer?
+  Integer(gets) rescue false
+end
```

Comment:
You've named this method `integer?` with a question mark, which implies that it should return a Boolean (`true` or `false`). However it looks like it returns the user's input as an integer. A better name for this method might be something like `parse_input`.